pageserver/src/page_service.rs

```diff
@@ -77,7 +77,7 @@ enum PagestreamBeMessage {
     Read(PagestreamReadResponse),
 }
 
-const HELLO_WORLD_ROW: BeMessage = BeMessage::DataRow(Bytes::from_static(b"hello world"));
+static HELLO_WORLD_ROW: BeMessage = BeMessage::DataRow(Bytes::from_static(b"hello world"));
```

Comment:
Clippy is mad because the `Bytes` implementation appears to allow interior mutability. This is mostly clippy being dumb (though it's not alone; even rustc will get upset if you try to take a const ref to a `const Bytes`). Easiest to just avoid the issue by using `static` instead.